cypress/jenkins/init.sh

```diff
@@ -164,6 +168,9 @@ fi
 
 echo "Rancher type: ${RANCHER_TYPE}"
 
+override_node=$(semver lt "${RANCHER_VERSION}" "2.9.99")
+if [[ ${override_node} -eq 0 && RANCHER_IMAGE_TAG != "head" ]]; then NODEJS_VERSION="16.20.2"; fi
```

Comment:
Do we need to add additional logic to use this node js version when using `v2.9-head`, `v2.8-head`, and `v2.7-head`?

Reply:
Rancher version is calculated using the head image. `helm search` is used to retrieve the latest Rancher chart version for that `-head` so we will end up with a `RANCHER_VERSION` 2.7.x up to 2.9.x compared to the highest minor Rancher chart version for 2.9, `2.9.99`.

Anything above here will land in either the `head` image that's excluded here or 2.10+
Those will use the node 20 configured in the job builder.

Reply:
ah that's right, there isn't any `*-head` for the helm charts. Cool, thank you!